### PR TITLE
Check registration using the Michigan Elections API

### PR DESCRIPTION
**Pull request by jacebrowning**
This isn't actually working. The client is making an `OPTIONS` request, not get.

### CODE REVIEW

File: web_client/app/infrastructure/api/api.ts [hunk 1]
```diff
@@ -19,7 +19,11 @@ class API {
     if (method != 'GET' && body) {
       request.body = JSON.stringify(body);
     }
-    return (fetch as any)(this.apiURL + url, request);
+    if (url.includes('://')) {
+      return (fetch as any)(url, request)
```
Comment (jacebrowning) [inline web_client/app/infrastructure/api/api.ts]: I assume we'll want to make a new file for the external API?

Comment (diiq) [reply web_client/app/infrastructure/api/api.ts]: I dunno that we need a whole new API class and file, but you're right that we definitely we don't want to do this, which sends the users' credentials to whatever URL gets plopped in there.

I'll take a look.

**Comment by diiq** [comment]
You're making a cross origin async request, so the browser must an OPTIONS request first. The Elections API will need to handle CORS OPTIONS requests, since it expects to be called exclusively as a 3rd party.

**Comment by jacebrowning** [comment]
👍 It's working!

**Pull request merged**